File: OneCodeTeam/Get-MAC-of-local-Ethernet-and-Wireless80211-17238/[VB]-Get MAC of local Ethernet and Wireless80211 (VBMACAddress)/VB/VBMACAddress/WMIHelper/WMIPropertyType.vb
'************************** Module Header ******************************'
' Module Name:  WMIPropertyType.vb
' Project:      VBMACAddress
' Copyright (c) Microsoft Corporation.
'
' The enum defines the WMI property types, which will be used to convert the 
' WMI property value to a .NET type. 
' 
' 
' This source is subject to the Microsoft Public License.
' See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
' All other rights reserved.
' 
' THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
' EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
' WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
'*************************************************************************'

Namespace WMIHelper
    Public Enum WMIPropertyType
        Bool
        ByteValue
        Int16
        Int32
        Int64
        UInt16
        UInt32
        UInt64
        DateTime
        StringValue
		ByteArray
		UInt16Array
		UInt32Array
		StringArray
		WMIObject
	End Enum
End Namespace

File: OneCodeTeam/Get-MAC-of-local-Ethernet-and-Wireless80211-17237/[C#]-Get MAC of local Ethernet and Wireless80211 (CSMACAddress)/C#/CSMACAddress/WMIHelper/WMIPropertyType.cs
/****************************** Module Header ******************************\
 * Module Name:  WMIPropertyType.cs
 * Project:      CSMACAddress
 * Copyright (c) Microsoft Corporation.
 *
 * The enum defines the WMI property types, which will be used to convert the 
 * WMI property value to a .NET type. 
 * 
 * 
 * This source is subject to the Microsoft Public License.
 * See http://www.microsoft.com/opensource/licenses.mspx#Ms-PL.
 * All other rights reserved.
 * 
 * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND, 
 * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED 
 * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
\***************************************************************************/
namespace CSMACAddress.WMIHelper
{
    public enum WMIPropertyType
    {
        Boolean,
        Byte,
        Int16,
        Int32,
        Int64,
        UInt16,
        UInt32,
        UInt64,
        DateTime,
        String,
        ByteArray,
        UInt16Array,
        UInt32Array,
        StringArray,
        WMIObject
    }
}
